### PR TITLE
Rashuai/bert cuda

### PR DESCRIPTION
**Pull request by RandySheriffH**
Add FastGelu as contrib op.

### CODE REVIEW

**Comment by wejoncy** [comment]
Is it possible to get the GPU device-id in custom op?

**Comment by RandySheriffH** [comment]
> Is it possible to get the GPU device-id in custom op?

No, custom op has no way to know the device yet.

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,6 +64,7 @@ option(OCOS_ENABLE_CV2 "Enable the operators in `operators/cv2`" ON)
 option(OCOS_ENABLE_VISION "Enable the operators in `operators/vision`" ON)
 option(OCOS_ENABLE_AUDIO "Enable the operators for audio processing" ON)
 option(OCOS_ENABLE_AZURE "Enable the operators for azure execution provider" OFF)
+option(OCOS_ENABLE_CONTRIB "Enable the contrib operators" ON)
```
Comment (wenbingl) [inline CMakeLists.txt:67]: This flag has dependency on CUDA, not sure how we can handle it correctly, and 'CONTRIB' is too general.
Maybe we just remove this flag, and only let USE_CUDA to enable it?

Comment (RandySheriffH) [reply CMakeLists.txt]: Dropped.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**